packages/phone-number-privacy/signer/src/signing/query-quota.ts

```diff
@@ -272,6 +282,33 @@ export async function getDollarBalance(logger: Logger, ...addresses: string[]):
   })
 }
 
+export async function getEuroBalance(logger: Logger, ...addresses: string[]): Promise<BigNumber> {
```

Comment:
Given that this is similar to `getDollarBalance` we can create a new method `getStableTokenBalance` that receives the token type as a parameter.

Reply:
Done